modules/react-firebase-docs/docs/react-firebase-auth/getting-started.md

```diff
@@ -100,6 +100,27 @@ Then use any of the other components anywhere in your component tree.
 </FirebaseAuthConsumer>
 ```
 
+`IfFirebaseAuthedAnd`
+
+```jsx
+<IfFirebaseAuthedAnd
+ filter={({ providerId, user }) => {
+   const email = get(user, "email", "");
```

Comment:
Can we do this example without lodash.get ? It would make it easier for readers to just copy paste the code and modify as they see fit

Reply:
yes sure will do!